app/data_collection/blog_data.py

```diff
@@ -0,0 +1,166 @@
+import re
+import requests
+from bs4 import BeautifulSoup
+from langchain_community.document_loaders import WebBaseLoader, AsyncHtmlLoader
+
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.firefox.service import Service
+from selenium.webdriver.firefox.options import Options
+import time
+
+# 크롤링을 할 때 사용하는 크롬 브라우저의 설정을 정의합니다.
+chrome_options = Options()
+chrome_options.add_argument("--headless")  # 브라우저를 헤드리스 모드로 실행합니다.
+chrome_options.add_argument("--disable-gpu")  # GPU를 사용하지 않도록 설정합니다.
+chrome_options.add_argument(
+    "--no-sandbox"
+)  # 샌드박스 모드를 사용하지 않도록 설정합니다.
+
+
+class WebsiteDataCrawler:
+    """Naver, NCSoft, Anthropic 블로그 사이트에서 블로그 콘텐츠를 크롤링해오는 클래스입니다
+
+    Attributes:
+        driver (selenium.webdriver): Javascript가 필요한 웹사이트를 크롤링할 때 필요한 webdriver입니다.
+    """
+
+    def __init__(self):
+        self.driver = webdriver.Firefox(options=chrome_options)
+        pass
+
+    def get_all_hrefs(self, url: str) -> list:
```

Comment:
타입형 힌트 작성시 더 구체적으로 작성 부탁드립니다🙏

```suggestion
    def get_all_hrefs(self, url: str) -> list[str]:
```

(이하 다른 함수들도 포함)

Reply:
타입 힌트 수정해서 [00d3a3b](https://github.com/f-lab-edu/llm-mlops/pull/2/commits/00d3a3b005cac27ff861a7d66e9bdb053c8181d0)에 올렸습니다!